proto/runtime_config.proto

```diff
@@ -206,63 +202,63 @@ message Network {
 // InterfacePriority for network interfaces
 message InterfacePriority {
 	// Name is the name of the network interface
-	optional string name = 1;
+	string name = 1;
 	// Priority for the interface
-	optional int64 priority = 2;
+	int64 priority = 2;
 }
 
 // Namespace is the configuration for a linux namespace
 message Namespace {
 	// Type is the type of Linux namespace
-	optional string type = 1;
+	string type = 1;
 	// Path is a path to an existing namespace persisted on disk that can be joined
 	// and is of the same type
-	optional string path = 2;
+	string path = 2;
 }
 
 // Device represents the information on a Linux special device file
 message Device {
 	// Path to the device.
-	optional string path = 1;
+	string path = 1;
 	// Device type, block, char, etc.
 	// TODO(vbatts) ensure int32 is fine here, instead of golang's rune
-	optional int32 type = 2;
+	int32 type = 2;
 	// Major is the device's major number.
-	optional int64 major = 3;
+	int64 major = 3;
 	// Minor is the device's minor number.
-	optional int64 minor = 4;
+	int64 minor = 4;
 	// Cgroup permissions format, rwm.
-	optional string permissions = 5;
+	string permissions = 5;
 	// FileMode permission bits for the device.
 	// TODO(vbatts) os.FileMode is an octal uint32
-	optional uint32 file_mode = 6;
+	uint32 file_mode = 6;
 	// Uid of the device.
-	optional uint32 uid = 7;
+	uint32 uid = 7;
 	// Gid of the device.
-	optional uint32 gid = 8;
+	uint32 gid = 8;
 }
 
 // Seccomp represents syscall restrictions
 message Seccomp {
 	// TODO(vbatts) string instead of "Action" type
-	optional string default_action = 1;
+	string default_action = 1;
 	repeated Syscall syscalls = 2;
 }
 
 // Syscall is used to match a syscall in Seccomp
 message Syscall {
-	optional string name = 1;
-	optional string action = 2;
+	string name = 1;
+	string action = 2;
 	repeated Arg args = 3;
 }
 
 // Arg used for matching specific syscall arguments in Seccomp
 message Arg {
```

Comment:
is this is seccomp/syscall specific, it might be better to have it as a nested message, ie this is now the type oci.Arg (same above for oci.Syscall), maybe oci.Seccomp.Syscall and oci.Seccomp.Syscall.Arg or oci.Seccomp.Arg